Helper/Data.php

```diff
@@ -288,9 +288,14 @@ public function formatAmount($amount, $currency)
                 $format = 2;
         }
 
+        if ($returnFormat) {
```

Comment:
I would outsource it to another function and would call that function in this function for formatting

Reply:
good idea, maybe better to do it in the PHP library and call it here

Reply:
yeah since we do the same in every plugin, okay then in the library :)

Reply:
not the responsibility of this method indeed. you can just split the responsibility. Are you adding this in the library?
Woul be great to add a task for it and update this to the library when we support al v49 PMs integration.